docs/maintain/maintain-guides-how-to-chill.md

```diff
@@ -64,12 +64,10 @@ different.
 
 ## Involuntary Chills
 
-If a validator was unresponsive for an entire session, the validator bond would be chilled in a
-process known as _involuntary chilling._ When a validator has been involuntarily chilled, it may
-restrict the validator from being selected in the next election depending on the session in which it
-was chilled (see considerations above). A chilled validator may re-declare the intent to validate at
-any time. However, it is recommended that the validator attempts to determine the source of the
-chill before doing so.
+Involuntary chilling restricts a validator from being selected in the next election depending on the
+session in which it was chilled (see considerations above). A chilled validator may re-declare the
+intent to validate at any time. However, it is recommended that the validator attempts to determine
+the source of the chill before doing so.
```

Comment:
Validators will no longer be chilled for offences. Prior to recent changes chilling was caused by ImOnline, Invalid Backing or Equivocations. ImOnline was removed altogether and involuntary chilling was completely disabled with the merge of https://github.com/paritytech/polkadot-sdk/pull/2226 (soon to be released).

Check this issue for a more comprehensive list of changes: https://github.com/orgs/paritytech/projects/119/views/15?pane=issue&itemId=61684472

Reply:
rm involuntary chill section